database/mssql/scripts/dbscripts/PSP_PIMS_S24_00/Alter Down/01_PSP_PIMS_S24_00_Alter_Down.sql

```diff
@@ -0,0 +1,3335 @@
+PRINT N'Drop primary key constraint dbo.PIMS_PRSCRC_H_PK'
+GO
+ALTER TABLE [dbo].[PIMS_PROPERTY_RESEARCH_FILE_HIST]
+	DROP CONSTRAINT [PIMS_PRSCRC_H_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.PRSCRC_PK
+PRINT N'Drop primary key constraint dbo.PRSCRC_PK'
+GO
+ALTER TABLE [dbo].[PIMS_PROPERTY_RESEARCH_FILE]
+	DROP CONSTRAINT [PRSCRC_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop table dbo.PIMS_PROPERTY_RESEARCH_FILE
+PRINT N'Drop table dbo.PIMS_PROPERTY_RESEARCH_FILE'
+GO
+DROP TABLE [dbo].[PIMS_PROPERTY_RESEARCH_FILE]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop unique constraint dbo.PIMS_RESRCH_H_UK
+PRINT N'Drop unique constraint dbo.PIMS_RESRCH_H_UK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_HIST]
+	DROP CONSTRAINT [PIMS_RESRCH_H_UK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.PIMS_RESRCH_H_PK
+PRINT N'Drop primary key constraint dbo.PIMS_RESRCH_H_PK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_HIST]
+	DROP CONSTRAINT [PIMS_RESRCH_H_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.RESRCH_PK
+PRINT N'Drop primary key constraint dbo.RESRCH_PK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE]
+	DROP CONSTRAINT [RESRCH_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.RSRCHS_PK
+PRINT N'Drop primary key constraint dbo.RSRCHS_PK'
+GO
+ALTER TABLE [dbo].[PIMS_RESEARCH_FILE_STATUS_TYPE]
+	DROP CONSTRAINT [RSRCHS_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.BCAJUR_PK
+PRINT N'Drop primary key constraint dbo.BCAJUR_PK'
+GO
+ALTER TABLE [dbo].[BCA_JURISDICTION]
+	DROP CONSTRAINT [BCAJUR_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.BCASAR_PK
+PRINT N'Drop primary key constraint dbo.BCASAR_PK'
+GO
+ALTER TABLE [dbo].[BCA_ASSESSMENT_AREA]
+	DROP CONSTRAINT [BCASAR_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.VOLUTY_PK
+PRINT N'Drop primary key constraint dbo.VOLUTY_PK'
+GO
+ALTER TABLE [dbo].[PIMS_VOLUME_UNIT_TYPE]
+	DROP CONSTRAINT [VOLUTY_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop sequence dbo.PIMS_RFILE_NUMBER_SEQ
+PRINT N'Drop sequence dbo.PIMS_RFILE_NUMBER_SEQ'
+GO
+DROP SEQUENCE [dbo].[PIMS_RFILE_NUMBER_SEQ]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_PROPERTY
+PRINT N'Alter table dbo.PIMS_PROPERTY'
+GO
+ALTER TABLE [dbo].[PIMS_PROPERTY] ALTER COLUMN [PROPERTY_AREA_UNIT_TYPE_CODE] nvarchar(20) NOT NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_PROPERTY]
+	ADD [PROPERTY_TENURE_TYPE_CODE] nvarchar(20) NOT NULL, 
+	[VOLUMETRIC_UNIT_TYPE_CODE] nvarchar(20) NULL, 
+	[PROPERTY_ANOMALY_TYPE_CODE] nvarchar(20) NULL, 
+	[PROPERTY_ROAD_TYPE_CODE] nvarchar(20) NULL, 
+	[PROPERTY_ADJACENT_LAND_TYPE_CODE] nvarchar(20) NULL
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Volumetric measurement unit of the parcel.' , 
```

Comment:
Huh, looking at the schema compare VOLUMETRIC_UNIT_TYPE_CODE changed to VOLU​ME_UNIT_​TYPE_COD​E which makes sense. However, this description got removed on the old field but not added on the new one from the looks of it.